src/geometry/ArborX_HyperSphere.hpp

```diff
@@ -47,15 +47,15 @@ struct Sphere
 
 } // namespace ArborX::ExperimentalHyperGeometry
 
-template <int DIM>
+template <int DIM, class FloatingPoint>
```

Comment:
I also made it a part of #918, but it does not matter.

Reply:
Wonder if triangle should be moved from `Experimental` to `ExperimentalHyperGeometry`.

Reply:
Sure.

Reply:
Dropped the changes to `HyperSphere`. They are not necessary here.